locales/index.js

```diff
@@ -0,0 +1,11 @@
+function getLocales(arr) {
+  return arr.map(function(item) {
+    return item.replace('./', '').replace('.json', '');
```

Comment:
This is mostly a copy paste job directly from donate. It's a good thing to consider making an npm repo for.